src/components/tools/Charts/Shared/Annotation/annotationUtils.js

```diff
@@ -0,0 +1,458 @@
+ * The Label is an absolutely positioned div. The position is calculated
+ * from the position property or data property. If neither property
+ * is present it will default to "top:0;left:0"
+ * @typedef {Object} Label
+ * @property {String} text – The text displayed in the label
+ * @property {Object} [data] – A row from data used in chart, used by layercake accessor functions to get x & y value
+ * @property {Object} [position] – Position of label using css props
+ * @property {Object} [style] - An object with list of css properties to overrise default label style
+ */
+
+/**
+ * The Connector object, used by the Annotation object
+ * The Connector is a svg path.
+ * @typedef {Object} Connector
+ * @property {("threshold"|"straight-arrow"|"swoopy-arrow")} type – allowed type of connector
+ * @property {("v"|"h")} [orientation = "v"] - indicates if threshold line is vertical or horizontal
+ * @property {Boolean} [clockwise = true] - indicates if swoopy arrow is clockwise or anticlockwise
+ * @property {Object} [source] - Position of connector end nearest to label
+ * @property {Object} [target] - Position of connector end farthest away from label
+ * @property {Object} [style] - An object with list of css properties to overrise default connector style
+ */
+
+/**
+ * The Source object, used by the Connector object
+ * @typedef {Object} Source
+ * @property {String} [anchor]  - attachment directives separated by a "-" (left/right-top/bottom/middle)
+ * @property {Object} [data] – A row from data used in chart, used by layercake accessor functions to get x & y value
+ * @property {Number} [dx] - number of pixels to offset x position
+ * @property {Number} [dy] - number of pixels to offset y position
+ */
+
+/**
+ * The Target object, used by the Connector object
+ * @typedef {Object} Target
+ * @property {Object} [data] – A row from data used in chart, used by layercake accessor functions to get x & y value
+ * @property {String} [x] - x position as number, pixel value or percentage
+ * @property {String} [y] - y position as number, pixel value or percentage
+ */
+
+/**
+ * The Data object, used by the Label & Connector objects
+ * @typedef {Object} Data
+ * @property {Date} [data.date = 0] - example for LineArea chart
+ * @property {Number} [data.value = 0] - example for LineArea chart
+ * @property {Number} [data.dx] - number of pixels to offset x position
+ * @property {Number} [data.dy] - number of pixels to offset y position
+ */
+
+/**
+ * The Position object, used by the Label object for absolute positioning
+ * @typedef {Object} Position
+ * @property {String} [position.top] - top edge as number, pixel value or percentage
+ * @property {String} [position.bottom] - bottom edge as number, pixel value or percentage
+ * @property {String} [position.left] - left edge as number, pixel value or percentage
+ * @property {String} [position.right] - right edge as number, pixel value or percentage
+ */
+export function getSampleAnnotations() {
```

Comment:
maybe add a line break here so that it's clear that the type definition for Position isn't describing the function below it.